scripts/exregional_run_prepstart.sh

```diff
@@ -107,14 +107,14 @@ YYYYMMDD=${YYYYMMDDHH:0:8}
 BKTYPE=0
 if [ ${cycle_type} == "spinup" ]; then
    echo "spin up cycle"
-  for cyc_start in ${CYCL_HRS_SPINSTART[@]}; do
+  for cyc_start in "${CYCL_HRS_SPINSTART[@]}"; do
     if [ ${HH} -eq ${cyc_start} ]; then
```

Comment:
Did you tested this?

The original form works on Jet and WCOSS. This may work but we need to test it make change.

Reply:
Yes, I did. Without the double quotes, it can not read the values I set in config.sh correctly when I was running the cold start. For example, I set CYCL_HRS_SPINSTART to be ( "00", "12") but the default values of ("HH1", "HH2") were passed to the script of prepcyc. Adding the double quotes should not stop it from working.